src/PhpSpreadsheet/Calculation/LookupRef/Helpers.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheet\Calculation\LookupRef;
+
+use PhpOffice\PhpSpreadsheet\Cell\AddressHelper;
+use PhpOffice\PhpSpreadsheet\Cell\Cell;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
+
+class Helpers
+{
+    private static function convertR1C1(string &$cellAddress1, ?string &$cellAddress2, bool $a1): string
+    {
+        if (!$a1) {
```

Comment:
Can we use explicit typed comparisons rather than lazy comparisons, the addition of class constants like 
```php
public const CELLADDRESS_USE_A1 = true;
public const CELLADDRESS_USE_R1C1 = true;
```
and similarly within the `INDIRECT()` implementation

This make it very obvious to anybody reading the code

These constants can then be used in the `INDIRECT()` function implementation as well

Reply:
Done.